testground/benchmark/benchmark/stateless.py

```diff
@@ -42,14 +42,17 @@ class CLI:
     def gen(
         self,
         outdir: str,
-        validators: int,
-        fullnodes: int,
         hostname_template=HOSTNAME_TEMPLATE,
-        num_accounts=10,
-        num_txs=1000,
-        config_patch={},
-        app_patch={},
+        options={},
     ):
+        print("options", options)
+        validators = options.get("validators")
+        fullnodes = options.get("fullnodes")
```

Comment:
```suggestion
        validators = options.get("validators")
        fullnodes = options.get("fullnodes")
```
```suggestion
        validators = options.get("validators", 3)
        fullnodes = options.get("fullnodes", 7)
```

since the options is optional, we should have a default for all parameters.